Api/Data/SDKTokenInterface.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * ClassyLlama_AvaTax
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @copyright  Copyright (c) 2018 Avalara, Inc.
+ * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+
+namespace ClassyLlama\AvaTax\Api\Data;
+
+interface SDKTokenInterface
+{
+    /**
+     * @return string
+     */
+    public function getToken();
+
+    /**
+     * @return int
+     */
+    public function getExpires();
+
+    /**
+     * @return string
+     */
+    public function getCustomer();
+
+    /**
+     * @return string
+     */
+    public function getClientId();
+
+    /**
+     * @return string
+     */
+    public function getSdkUrl();
+}
```

Comment:
SourceTree warns me when committing files without newlines at the bottom. Please configure your Git GUI or CLI to do the same. All files should have an empty newline at the bottom.